client/scss/components/_slider.scss

```diff
@@ -27,13 +23,7 @@
 
 .slider-control--prev {
   .control-arrow {
-    transform: rotate(90deg);
-  }
-}
-
-.slider-control--next {
-  .control-arrow {
-    transform: rotate(-90deg);
+    transform: rotate(180deg);
```

Comment:
Didn't we have a flip class somewhere?

Reply:
We have icon rotating modifiers (e.g. `icon--180`), but nothing more generic I don't think.